drivers/virtualbox/virtualbox.go

```diff
@@ -499,7 +498,7 @@ func (d *Driver) Kill() error {
 }
 
 func (d *Driver) GetState() (state.State, error) {
-	stdout, stderr, err := vbmOutErr("showvminfo", d.MachineName,
+	stdout, stderr, err := d.queryVirtualBox("showvminfo", d.MachineName,
```

Comment:
As mentioned above, I know it's nitpicky, but my preference would be to have this as `d.vbmOutErr(`for now.